rclcpp/test/rclcpp/test_timer.cpp

```diff
@@ -283,3 +298,12 @@ TEST_F(TestTimer, test_failures_with_exceptions)
       std::runtime_error("Timer could not get time until next call: error not set"));
   }
 }
+
+/// Simple test of a timer without autostart
+TEST_F(TestTimer, test_timer_without_autostart)
+{
+  EXPECT_TRUE(timer_without_autostart->is_canceled());
+  // Reset to change start timer
+  timer_without_autostart->reset();
+  EXPECT_FALSE(timer_without_autostart->is_canceled());
```

Comment:
we should also check that this timer will now trigger

Reply:
Done.